drafts/use-cases/1.entry-point.md

```diff
@@ -0,0 +1,108 @@
+## Entry point
+
+### Story
+As an application user
+I want my application to consume an entry point for the Web API by providing a base URL
+So I can start managing my events.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var data = client.get("/api");
```

Comment:
>Why not? If an API documentation exists, it should be referenced form any resources that is part of the API...
>and, if possible, also from every other resource the API publisher controls.
Maybe this requirement should be added to the spec? I believe we had a short discussion on expressing what client must, should and may understand.